mopidy/core/library.py

```diff
@@ -227,9 +227,11 @@ def lookup(self, uri=None, uris=None):
         results = {u: [] for u in uris}
 
         # TODO: lookup(uris) to backend APIs
-        for backend, backend_uris in self._get_backends_to_uris(uris).items():
-            for u in backend_uris:
-                futures[(backend, u)] = backend.library.lookup(u)
+        futures = {
```

Comment:
I find this a lot harder to read and parse with the multiple for loops in this expansion. To the point of it being hard to follow what the real change is.